### DISCUSSION TITLE
[BUG] Problemas en la configuración de Reports

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#282] [4. Bug Reports] [answered]
Hola 

Comento:

Resulta que quería configurar un usuario con rol **Reports** (report-*-write / report-*-read).

El tema es que al dejar eso con el usuario, puede ver las secciones (los títulos de los reportes), pero no puede ver el contenido de éstos como se adjunta la imagen:

![image](https://user-images.githubusercontent.com/61419471/122618545-7432ff80-d05c-11eb-989b-d0ff987e0242.png)

Para que el usuario pueda ver el contenido, requiere tener el permiso: **web-admin-write**, el problema es al tener ese permiso, el usuario podrá ver el Web Admin completo y no debería tener acceso.

Por lo que necesito que si pueden arreglar los permisos de reportes, para que el usuario pueda ver sin problemas y no tener el web admini.

Me apoyaron @gmoplaza y @agallardol .

Gracias =>

**Comment by NicolasPLG** [comment]
@jdurandpaz 4UI. Debería ser que el usuario no deba ver el admin, porque podría realizar algún cambio.

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG se creó el ticket #3101 .

Actualmente es un ticket de baja prioridad, si necesitas priorizarlo por favor contacta a tu representante de Cotalker.
